service/share/share.go

```diff
@@ -9,18 +9,20 @@ import (
 	format "github.com/ipfs/go-ipld-format"
 	logging "github.com/ipfs/go-log/v2"
 	"github.com/ipfs/go-merkledag"
-
+	"github.com/tendermint/tendermint/pkg/consts"
 	"github.com/tendermint/tendermint/pkg/da"
 
 	"github.com/celestiaorg/celestia-node/ipld"
 	"github.com/celestiaorg/celestia-node/ipld/plugin"
 	"github.com/celestiaorg/nmt"
 	"github.com/celestiaorg/nmt/namespace"
-	"github.com/celestiaorg/rsmt2d"
 )
 
 var log = logging.Logger("share")
 
+// DefaultCodec sets the default rsmt2d.Codec for shares.
+var DefaultCodec = consts.DefaultCodec
```

Comment:
Do we need to export this too?

Reply:
Yes

Reply:
I don't know why but ok

Reply:
Well actually we can make itpublic. The case I was thinking can be done separately. Will do it privately.